WooCommerce/Classes/ViewRelated/Orders/Order Details/Address Edit/EditOrderAddressForm.swift

```diff
@@ -60,7 +60,7 @@ final class EditOrderAddressHostingController: UIHostingController<EditOrderAddr
 
                 switch notice {
                 case .success:
-                    self?.systemNoticePresenter.enqueue(notice: .init(title: EditOrderAddressForm.Localization.success, feedbackType: .error))
+                    self?.systemNoticePresenter.enqueue(notice: .init(title: Localization.success, feedbackType: .error))
```

Comment:
We can't easily reference nested declarations from `EditOrderAddressForm` when it has generic requirements:
> Static stored properties not supported in generic types

So I flattened constants to private declarations at top level.